src/components/Card/Card.styled.ts

```diff
@@ -1,15 +1,78 @@
 import styled from 'styled-components';
 
-export const CardContainer = styled.div<{hasNavigation: boolean}>`
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
-    border: 1px solid black;
-    background: #ddd;
-    padding: 20px;
-    width: 250px;
-    max-height: 200px;
-    cursor: ${props => (props.hasNavigation ? 'pointer' : 'default')};
-    margin: 5px;
+export const CardContainer = styled.div<{hasNavigation: boolean, isUser: boolean, isLeader: boolean}>`
+  margin: 10px;
+  background-color: white;
+  border-radius: 10px;
+  box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+  overflow: hidden;
+  margin-bottom: 20px;
+  text-align: center;
+  cursor: ${props => (props.hasNavigation ? 'pointer' : 'default')};
+  display: ${props => (props.isUser ? 'grid' : 'flex')};
+  justify-content: center;
+  align-items: center;
+  
```

Comment:
Using conditional styles to get a better visual on the Card when the user is a Team Leader or not, and if it is a user